### PR TITLE
Mihai/wp plugin

### PR DESCRIPTION
**Pull request by netdesignr**
Hi @gtvj,

I've added stuff to the README file. Can you please review it? Do you think that this covers it all?

Changes from -> Add instructions to README -< and down

Thanks a lot!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,74 @@
...
 
-- [Project findings](https://github.com/nationalarchives/ds-cookie-consent/blob/main/docs/about-the-project.md)
+## Installation
+- Upload, install and activate the plugin
+
+## Create Cookies Settings page
+
+- Add shortcode [ds-cookie-consent-settings] to the website
+
+- Content for the cookie sections ca be added from TNA Cookie Consent dashboard settings page
```
Comment (gtvj) [inline README.md]: I think there's a typo here: `ca` instead of `can`

File: README.md [hunk 2]
```diff
@@ -1,5 +1,74 @@
...
+
+## Banner
+<img width="1263" alt="Screenshot 2021-04-20 at 10 35 43" src="https://user-images.githubusercontent.com/5245264/115373917-3b91b980-a1c4-11eb-82f7-96acab4a45cb.png">
+
+
+### HTML
+
+[https://github.com/nationalarchives/ds-cookie-consent/blob/develop/includes/utils.php#L20-L31](Banner)
```
Comment (gtvj) [inline README.md]: Not sure if this is expected but I'm getting a 404 for this page.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,74 @@
...
+
+### JavaScript
+
+[https://github.com/nationalarchives/ds-cookie-consent/blob/develop/src/api/dsCookieConsentBannerAPI.js](API)
```
Comment (gtvj) [inline README.md]: Not sure if this is expected but I'm getting a 404 for this page.

File: README.md [hunk 4]
```diff
@@ -1,5 +1,74 @@
...
+
+### CSS
+
+[https://github.com/nationalarchives/ds-cookie-consent/blob/develop/dist/ds-cookie-consent.css](Cascading_Stylesheet)
```
Comment (gtvj) [inline README.md]: Not sure if this is expected but I'm getting a 404 for this page.

File: README.md [hunk 5]
```diff
@@ -1,5 +1,74 @@
...
+
+### Plugin init
+
+[https://github.com/nationalarchives/ds-cookie-consent/blob/mihai/wp_plugin/src/ds-cookie-consent.js](Initialise_the_plugin)
```
Comment (gtvj) [inline README.md:33]: Not sure if this is expected but I'm getting a 404 for this page.

File: README.md [hunk 6]
```diff
@@ -1,5 +1,74 @@
...
+
+## External usage instructions
+
+- Copy over the HTMl banner
```
Comment (gtvj) [inline README.md]: I think there's a typo here - should it be an uppercase 'L'?

**Review by gtvj — CHANGES_REQUESTED** [review]
Hi Mihai, 

As requested, **I've limited this review to the README.md file because other files are being dealt with separately**. 

I've added a few comments. If you could address these I'd be happy to merge.

File: README.md (hunk 6 above)
Comment (netdesignr) [reply README.md]: Done

File: README.md (hunk 5 above)
Comment (netdesignr) [reply README.md:33]: Done

File: README.md (hunk 4 above)
Comment (netdesignr) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (netdesignr) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (netdesignr) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (netdesignr) [reply README.md]: Done

**Comment by gtvj** [comment]
Merging this now. The review has only been on the README because I understand other files are being dealt with separately.

**Pull request merged**